src/ansible/inventory/inventory.yml

```diff
@@ -1,8 +1,8 @@
 all:
   vars:
     ansible_python_interpreter: /usr/bin/python3
-    remote_user: root
-    ansible_ssh_user: root
+    remote_user: ubuntu
```

Comment:
What if I use RedHat?

Reply:
Maybe a more generic user would be preferable? wdyt?

Reply:
Or maybe we should focus on a small set of supported distros ? (Ubuntu and Debian at first)

Or both? (generic user + small set of distros)

Reply:
> Maybe a more generic user would be preferable? wdyt?

you are right, that was intended to fix the aws issues, before the Ubuntu idea :)

Reply:
> you are right, that was intended to fix the aws issues, before the Debian idea :)

btw I'm doing some tests on a single node and except some already known bugs and some missing parts in the doc it works very well :) :+1:  :heart: